src/test/java/io/openshift/boosters/OpenshiftIT.java

```diff
@@ -182,7 +182,13 @@ private void scale(String name, int replicas) {
                     .withLabel("deploymentconfig", name)
                     .list()
                     .getItems();
-            return pods.size() == replicas && pods.stream().allMatch(Readiness::isPodReady);
+            try {
+                return pods.size() == replicas && pods.stream().allMatch(Readiness::isPodReady);
+            } catch (IllegalStateException e) {
+                // the 'Ready' condition can be missing sometimes, in which case Readiness.isPodReady throws an exception
```

Comment:
There's an open pull request to add this method: https://github.com/openshiftio/booster-common/pull/6, I'll ask @RadekKoubsky to update it with this change.